prov/verbs/src/fi_verbs.c

```diff
@@ -586,21 +586,24 @@ static int fi_ibv_fi_to_rai(const struct fi_info *fi, uint64_t flags, struct rdm
 	if (!fi)
 		return 0;
 
-	switch(fi->addr_format) {
-	case FI_SOCKADDR_IN:
-		rai->ai_family = AF_INET;
-		break;
-	case FI_SOCKADDR_IN6:
-		rai->ai_family = AF_INET6;
-		break;
-	case FI_SOCKADDR_IB:
-		rai->ai_family = AF_IB;
-		break;
-	case FI_SOCKADDR:
-	case FI_FORMAT_UNSPEC:
-		break;
-	default:
-		FI_INFO(&fi_ibv_prov, FI_LOG_CORE, "Unknown fi->addr_format\n");
+	if (fi->addr_format) {
+		rai->ai_flags |= RAI_FAMILY;
```

Comment:
If the addr_format is FI_SOCKADDR and if fi->src_addr or fi->dst_addr is available, we would have to examine the sa_family field to find the correct address family, right? In that case RAI_FAMILY would be valid once we set rai->ai_family correctly.